stages/org.osbuild.dnf.config

```diff
@@ -6,7 +6,8 @@ The stage changes persistent DNF configuration on the filesystem. Currently,
 only DNF variables can be defined.
 """
 
-
+import configparser
```

Comment:
I wonder if we should use that `iniparse` parser, which will better preserve formatting (and comments I think). See `stages/org.osbuild.rhsm` stage.

Reply:
Yea configparser also adds an extra newline at the end :/

Reply:
I agree that `iniparse` is better if we need to read an existing INI config. If we just need to create one, it would probably not matter.